apps/client/lib/get-data-api.ts

```diff
@@ -10,6 +10,7 @@ export async function getAllUsers(): Promise<User[]> {
   const query = queries.GET_USERS
```

Comment:
added basic data validation to all queries, added cache-clear to `getFullReport`